.github/workflows/lint-test-chart.yaml

```
@@ -18,6 +18,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Run helm values.schema.json check
+        uses: losisin/helm-values-schema-json-action@v1
```

```suggestion
        uses: losisin/helm-values-schema-json-action@72a011e2ba3fa31880588d54b715779aaf99270f # v1.5.8
```
GitHub actions should be referenced by digest.

---

Sry I'm not using gihtub action. It works fine when chart is in root, not with current project structure https://github.com/kubernetes-sigs/external-dns/pull/5075#issuecomment-2653029648

Do you still prefer action vs custom script?

---

I've created an issue https://github.com/losisin/helm-values-schema-json-action/issues/318. Currently there is a poor support when chart is in `charts/....` folder and output contains no diff

---

You can just use a scripts like we do for `helm-docs`.

---

Updated